### PR TITLE
SalesTax: Remove Goodie

### PR DESCRIPTION
**Pull request by MrChrisW**
Completing: #960 - Remove SalesTax Goodie

+ SalesTax Goodie has been replaced by a Spice in [#1474](https://github.com/duckduckgo/zeroclickinfo-spice/pull/1474)

CC// @moollaza

### CODE REVIEW

File: lib/DDG/Goodie/SalesTax.pm [hunk 1]
```diff
@@ -1,58 +0,0 @@
-package DDG::Goodie::SalesTax;
-#ABSTRACT: Returns the sales tax for any state (not including territories) in the United States. 
-use DDG::Goodie;
-use Locale::SubCountry;
```
Comment (mintsoft) [inline lib/DDG/Goodie/SalesTax.pm]: Can this be removed from `dist.ini` or do other goodies depend on it?

Comment (MrChrisW) [inline lib/DDG/Goodie/SalesTax.pm]: Good thinking, looks like the CurrencyIn Goodie uses it https://github.com/duckduckgo/zeroclickinfo-goodies/blob/mastee/lib/DDG/Goodie/CurrencyIn.pm#L21

**Comment by mintsoft** [comment]
Fair enough!

**Comment by moollaza** [comment]
:+1:

**Pull request merged**